core/descriptors/src/org/jetbrains/kotlin/renderer/DescriptorRenderer.kt

```diff
@@ -221,6 +221,7 @@ interface DescriptorRendererOptions {
     var renderConstructorKeyword: Boolean
     var renderUnabbreviatedType: Boolean
     var includeAdditionalModifiers: Boolean
+    var includeSuspendModifier: Boolean
```

Comment:
Are there any cases when this option should be false?

Reply:
I'm not sure, to be honest. `OverridesCompletion.PRESENTATION_RENDERER` used to omit `suspend` but that's what the PR fixes. I could render `suspend` unconditionally if you think that's the correct behavior.

Reply:
Yes, I think it'd be best to always render `suspend` because I don't see why it would be useful not to do that

Reply:
I removed the property so that `suspend` is always rendered when applicable.